solutions/api/src/lambda/token/utils/verifyJti.ts

```diff
@@ -0,0 +1,45 @@
+import assert from "node:assert";
+import { getDynamoDbClient } from "../../../../../commons/utils/awsClient/dynamodbClient/index.js";
+import { errorManager } from "./errors.js";
+
+export const verifyJti = async (jti: string | undefined) => {
+  let item;
+  const dynamoDbClient = getDynamoDbClient();
+  assert.ok(
+    process.env["REPLAY_ATTACK_TABLE_NAME"],
+    "REPLAY_ATTACK_TABLE_NAME not set",
+  );
+  const tableName = process.env["REPLAY_ATTACK_TABLE_NAME"];
+
+  if (!jti) {
+    errorManager.throwError("invalidRequest", "jti is missing");
+    return;
+  }
+
+  try {
```

Comment:
This could be done with a single DB call.  create a PutItem statement with a condition that `nonce` does not exist with same value.  If a conditional error occurs throw `jti found`.  [PutItem and ConditionalCheckFailedException] -

```
TableName: tableName
ConditionExpression: attribute_not_exists("nonce")
...
 } catch (error: any) {
    if (error.name === "ConditionalCheckFailedException") {
      throw invalidReqesut error
    } else {
      throw server error
    }
}
```